setup.py

```diff
@@ -5,9 +5,9 @@
     long_description = readme.read()
 
 setup(
-    name='edx-ecommerce-api-client',
+    name='edx-rest-api-client',
     version='1.1.2',
```

Comment:
The version number needs to change so that there is no conflict with the old name+version combo.